src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -1,23 +1,24 @@
 ' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+Imports System.Collections.Concurrent
 Imports System.Collections.Immutable
-Imports System.Runtime.CompilerServices
 Imports Microsoft.CodeAnalysis.VisualBasic
 
 Namespace Microsoft.CodeAnalysis.Semantics
-    Partial Friend NotInheritable Class VisualBasicOperationFactory
```

Comment:
>NotInheritable [](start = 19, length = 14)

I am not sure why ```NotInheritable``` modifier got removed. Do we intend to inherit from this class now?